qiskit/_result.py

```diff
@@ -0,0 +1,211 @@
+from qiskit._qiskiterror import QISKitError
+
+class Result(object):
+    """ Result Class.
+
+    Class internal properties.
+
+    Methods to process the quantum program after it has been run
+
+    Internal::
+
+        qobj =  { -- the quantum object that was complied --}
+        result = {
+            "status": --status (string),
+            "result":
+                [
+                    {
+                    "data":
+                        {  #### DATA CAN BE A DIFFERENT DICTIONARY FOR EACH BACKEND ####
+                        "counts": {’00000’: XXXX, ’00001’: XXXXX},
+                        "time"  : xx.xxxxxxxx
+                        },
+                    "status": --status (string)--
+                    },
+                    ...
+                ]
+            }
+    """
+
+    def __init__(self, qobj_result, qobj):
+        self.__qobj = qobj
+        self.__result = qobj_result
+
+    def __str__(self):
+        """Get the status of the run.
+
+        Returns:
+            the status of the results.
+        """
+        return self.__result['status']
+
+    def __iadd__(self, other):
+        """Append a Result object to current Result object.
+
+        Arg:
+            other (Result): a Result object to append.
+        Returns:
+            The current object with appended results.
+        """
+        if self.__qobj['config'] == other.__qobj['config']:
+            if isinstance(self.__qobj['id'], str):
+                self.__qobj['id'] = [self.__qobj['id']]
+            self.__qobj['id'].append(other.__qobj['id'])
+            self.__qobj['circuits'] += other.__qobj['circuits']
+            self.__result['result'] += other.__result['result']
+            return self
+        else:
+            raise QISKitError('Result objects have different configs and cannot be combined.')
+
+    def __add__(self, other):
+        """Combine Result objects.
+
+        Note that the qobj id of the returned result will be the same as the
+        first result.
+
+        Arg:
+            other (Result): a Result object to combine.
+        Returns:
+            A new Result object consisting of combined objects.
+        """
+        ret = copy.deepcopy(self)
```

Comment:
Can you include an `import copy` statement on this file? It seems it was lost during the moving of the `Result` class to its own module.